code/modules/food&drinks/drinks/drinks/bottle.dm

```diff
@@ -7,6 +7,7 @@
 /obj/item/weapon/reagent_containers/food/drinks/bottle
 	amount_per_transfer_from_this = 10
 	volume = 100
+	throwforce = 15
```

Comment:
Breaks after one throw though, and that's how much it does in melee (breaks after one hit in melee as well)